app/containers/message/index.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { Keyboard, ViewStyle } from 'react-native';
+import { Keyboard, ViewStyle, Animated } from 'react-native';
```

Comment:
Why would you use stock RN's Animated instead of reanimated?

Reply:
Ok I will use Animated from 'react-native-reanimated'